src/storages/types.ts

```diff
@@ -355,6 +360,15 @@ export interface IImpressionCountsCacheSync extends IRecorderCacheProducerSync<R
   pop(toMerge?: Record<string, number> ): Record<string, number> // pop cache data
 }
 
+export interface IImpressionCountsCacheInRedis extends IImpressionCountsCacheSync {
+  postImpressionCountsInRedis(body: string): Promise<[Error | null, any][]>
```

Comment:
This method (`postImpressionCountsInRedis`) is actually private (only used inside ImpressionCountsCacheInRedis), and since we provide it to the `setInterval` it should not return a promise. 

In summary, we can remove this interface.